helm/image_templating.md

```diff
@@ -0,0 +1,17 @@
+# Helm Chart Image Templating
+
+## Templating
+
+When templating container images in charts `values.yaml` should be structured
+like this. This allows us to override `image.registry` in some control planes
+e.g. AWS China.
+
+
+```yaml
+image:
+  registry: quay.io
```

Comment:
Not related to this change but most of the charts are simply `quay.io` images, can we may set up the default value for that?

Reply:
Yes we use `quay.io` everywhere except AWS China where we use `registry-intl.cn-shanghai.aliyuncs.com`.

But at some point we may want to run our own registry like Harbor and for some onprem control planes as well as China.